assignments/002-unix-intro/unix-intro.sh

```diff
@@ -0,0 +1,593 @@
+@K00364:64:HTYYCBBXX:1:1101:1824:48192/2
+CACAAGGAACGACAGAGGGTAGTGCGTACGGCCCAGCACATCACTGGGACCAAGCATCCTGCAATCCAGGACCTCCATACCAGGCGGTGTCAGAGGAAGGCCCTAAAATTGTCAAAGACTCCAGCCACCCTAGTCATTAACTCTTCGCTCT
++
+A<-A-<-F<AJ-<---<<--<-<-7FFJ-77A<-<--7-<7-<-7<J<77A<--7-AAA7-AF-7FAF<-<FJ---7-77FJA7-77F7FFJ7F<--<--77)A7FA-A7-<---7<<F7-<-7<))7--)<)7<AF----7<<<<-)-7)
+@K00364:64:HTYYCBBXX:1:1101:2869:22080/2
+CTATGATCAACAGCGTTTTGTGATTTACCCCCGTGATGCACTCACTGGTATGTCTGAACAGCTTGAGGCCACATCTAGGGTTGCCAGACAGAATAGACTTGCTTTGGATATGCTTCTTGCCAGTCAGGGGGGTGTCTGTAAGATGTTCGGT
++
+AAAFAFJJJJJJJJJJ-FJJJJJJFJJJJFJJJJJJJJJJJJJFJFFJFFFFFFJFFJJJFJFAAA777AAJJJAA--A<-A<-JJA-<JFJJJ-7<-AAJJFJJJ<JJ<JAFFFF---7<<7777<)7<<-<777-<-7--<FJJ<F-<<
+
+#### Ex. 33: What do you notice about the names of the two reads in each 
+####         file (Lines 1 and 5, that start with a `@`)
+
+They are both the same K00364. 
+
+#### Ex. 34: Print the date, redirect it to a file called `now.txt`,
+####         catenate that file to _stdout_ and then remove it
+
+(base) [zhimei@colostate.edu@login11 002-unix-intro]$ date +%D
+01/29/24
+(base) [zhimei@colostate.edu@login11 002-unix-intro]$ date +%D > n
+ow.txt
+(base) [zhimei@colostate.edu@login11 002-unix-intro]$ cat now.txt
+01/29/24
+(base) [zhimei@colostate.edu@login11 002-unix-intro]$ rm now.txt
+
+#### Ex. 35: Count the number of lines/words/characters (using `wc`)
+####         in the two SAM files in the `sam` directory
+
+(base) [zhimei@colostate.edu@login11 sam]$ wc DPCh_plate1_A05_S5.sam
+ 14219  45092 511203 DPCh_plate1_A05_S5.sam
+ (base) [zhimei@colostate.edu@login11 sam]$ wc DPCh_plate1_A06_S6.sam
+ 14150  43985 479977 DPCh_plate1_A06_S6.sam
+
+#### Ex. 36: Note that gzcat will decompress all files you give itand send the 
+####         output to _stdout_. Knowing this, count how many lines are in each of 
+####         the `R1` files in `fastq` (after decompressing them into text files),
+####         and also count up all the lines in the `R1` files in `fastq` by catting
+####         all the .gz files together and the gzcatting it into wc
+####         Note they should be the same.
+
+  3333  20035 915507 total
+  3333  20035 915507 total
+  
+#### Ex. 37: Now, gzcat all the `R1` files in `fastq` and redirect that
+####         to a file called `R1_all_via_gzcat.fq` in the top directory of the repo
+
+(base) [zhimei@colostate.edu@login11 fastq]$ touch R1_all_via_gzcat.fq
+(base) [zhimei@colostate.edu@login11 fastq]$ zcat *R1.fq.gz > R1_all_via_gzcat.fq
+
+#### Ex. 38: People are usually familiar with using `cat` to catenate text
+####         files. It turns out that you can also catenate `gzip` compressed files. 
+####         The result is another properly gzipped file that can be decompressed. 
+####         Catenate all the gzipped `R1` files in `fastq` into a single (still gzipped)
+####         file called `R1_all_via_cat.fq.gz`
+
+(base) [zhimei@colostate.edu@login11 fastq]$ touch R1_all_via_cat.fq.gz
```

Comment:
You don't need to touch the file before redirecting into it.  You can just redirect into a file that doesn't even already exist.